.github/workflows/preview.yaml

```
@@ -67,9 +67,24 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Run Cypress
+      - name: Download snapshots from artifacts
+        run: |
+          mkdir -p cypress/snapshots
+        uses: actions/download-artifact@v3
```

actionlint (1.7.4)

this step is for running shell command since it contains at least one of "run", "shell" keys, but also contains "uses" key which is used for running action

(syntax-check)